spec/spec_helper.rb

```diff
@@ -55,9 +56,9 @@ def generate_topic_name
     "#{RUN_ID}-topic-#{@@topic_number}"
   end
 
-  def create_random_topic(*args)
+  def create_random_topic(**args)
     topic = generate_topic_name
-    create_topic(topic, *args)
+    create_topic(topic, **args)
```

Comment:
These changes were needed to support ruby 3 due to https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/.